pkg/operator/k8sutil/network_test.go

```diff
@@ -53,12 +73,15 @@ func TestNetwork_ApplyMultusJSON(t *testing.T) {
 	}
 
 	objMeta := metav1.ObjectMeta{}
+	objMeta.Labels = map[string]string{
+		"app": "rook-ceph-osd",
+	}
 	err := ApplyMultus(net, &objMeta)
 	assert.NoError(t, err)
 
-	assert.Contains(t, objMeta.Annotations, "k8s.v1.cni.cncf.io/networks")
-	assert.Contains(t, objMeta.Annotations["k8s.v1.cni.cncf.io/networks"], `{"name": "macvlan", "interface": "net1"}`)
-	assert.Contains(t, objMeta.Annotations["k8s.v1.cni.cncf.io/networks"], `{"name": "macvlan", "interface": "net2"}`)
+	want := map[string]string{"k8s.v1.cni.cncf.io/networks": `[{"name": "macvlan", "interface": "net1"}, {"name": "macvlan", "interface": "net2"}]`}
+
+	assert.Exactly(t, objMeta.Annotations, want)
```

Comment:
@BlaineEXE will have to use `assert.Contains()` as splitting the strings will get complex.
The unit test fails if the order of elements change in the map value string.